vignettes/LPWC.Rmd

```diff
@@ -135,7 +135,7 @@ This is only relevant for the low penalty because the high penalty uses a single
 
 ## Obtaining gene-gene similarities
 
-LPWC computes similarity (an adjusted weighted correlation) between genes and returns three outputs: correlation of genes in dist form (see as.dist), the best lag used for each gene, and the C used.
+LPWC computes similarity (an adjusted weighted correlation) between genes and returns three outputs: correlation of genes in dist form (see as.dist), the best lag used for each gene, and the C used. The lags introduced are in reference to the original time points (i.e. lag of 0). 
```

Comment:
I don't think this is enough of an explanation.  Should we add Figure 2 from our paper to the vignette to explain the lag concept and how the lags refer to the original timepoints?  If not that, that we should use the example timepoint vector `0, 2, 4, 6, 8, 18, 24, 32, 48, 72` to illustrate what positive and negative lags mean.

Reply:
@thevaachandereng, apologies if I'd confused you. I'll try to break down my understanding of the function as follow:
  1) suppose we have 5 genes, all sharing 1 set of common timepoint
  2) suppose `bestlag()` yields c(-1, 1, 0, 2, 0)
  3) since we have 2 zero elements (i.e., 3rd and 5th), do those elements with non-zero lags (i.e., 
      1st, 2nd and 4th) lagged with reference to the 3rd or 5th element?